src/ui/public/markdown/_github_markdown.scss

```diff
@@ -0,0 +1,462 @@
+// The MIT License (MIT)
```

Comment:
Did this file change significantly? Only reason I ask is that this might be a file we don't actually want to variables (at least yet). Likely the sizing changes could break scrolling limits on panels within user-created dashboards. We might just want to give ourselves a todo to change the sizing in 7.0.

I don't have any big opinions on it, just thinking it through.

Reply:
I've kept all the measurements the same. They're based on a base font size and use `em`'s to scale instead of `rem`.

Example: (left side is this PR, right side is demo site)
<img width="825" alt="screen shot 2018-11-27 at 12 39 48 pm" src="https://user-images.githubusercontent.com/549577/49100405-d713df80-f241-11e8-9251-05a2bc05e6d8.png">

Reply:
Headings are slightly bolder because it does use the EUI bold variable, and some coloring is more grayscale, but that's really the main differences.

Reply:
Excellent.